pylint_odoo/checkers/no_modules.py

```diff
@@ -538,6 +545,17 @@ def visit_dict(self, node):
                 self.add_message('resource-not-exist', node=node,
                                  args=(key, resource))
 
+        # Check if the website is valid URI
+        website = manifest_dict.get('website', '')
```

Comment:
Reduce the cyclomatic complexity here a little using by default `manifest_dict.get('website') or 'https://'` and avoid nested `if`